Gems/AtomTressFX/Code/Components/HairSystemComponent.h

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+#pragma once
+
+#include <AzCore/Component/Component.h>
+#include <AzCore/Asset/AssetCommon.h>
+
+#include <Atom/RPI.Public/Pass/PassSystemInterface.h>
+
+namespace AZ
+{
+    namespace Render
+    {
+        class SharedBuffer;
+
+        namespace Hair
+        {
+            class HairSystemComponent final
+                : public Component
+            {
+            public:
+                AZ_COMPONENT(HairSystemComponent, "{F3A56326-1D2F-462D-A9E8-0405A76601A5}");
```

Comment:
Nit: RTTI macros should have the namespace in front of the class.

Reply:
As before. If it becomes consensus across the board I will gladly change